app/models/work_package.rb

```diff
@@ -508,7 +502,6 @@ def estimated_hours=(h)
     converted_hours = (h.is_a?(String) ? h.to_hours : h)
     write_attribute :estimated_hours, !!converted_hours ? converted_hours : h
   end
-  # <<< issues.rb <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
 
   # Safely sets attributes
   # Should be called from controllers instead of #attributes=
```

Comment:
The `safe-attributes=` [method](https://github.com/opf/openproject/blob/dev/app/models/work_package.rb#L518)